content/installation/net/configuration/YAML-properties-net.md

```diff
@@ -61,6 +61,10 @@ All properties in this section must be put under the `agent` node, as shown in t
 
 * **agent**:
 
+#### Auto-update
+
+Use the following properties to control auto-update behavior of the .NET Framework Agent.  These settings do not apply to the .NET Core agent, which does not have the capability to auto-update.
+
```

Comment:
Is the original .NET agent officially changing names to ".NET Framework"?

Reply:
Well don't know about "official name change" but I think we shouldn't say just say ".NET Agent" in the document text anymore.  We should be more specific to be less ambiguous.  Maybe 

I would suggest we only have the following 3 names:

.NET Framework Agent - when referring only to the current agent
.NET Core Agent - when referring only to the upcoming .NET Core agent
.NET Agents - when referring to both or either agent

@DanFiedler can chime in here.